lib/src/link/Link.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import styled, { ThemeProvider } from "styled-components";
-import PropTypes from "prop-types";
 import { spaces } from "../common/variables.js";
 import useTheme from "../useTheme.js";
+import Overload, { LinkTextProps, LinkIconProps, LinkCommonProps } from "./types";
```

Comment:
Remove `LinkCommonProps` from the import.

Reply:
Changed done.